test/index.html

```diff
@@ -49,79 +49,146 @@
 
-  <hr/>
+// Close the dropdown if the user clicks outside of it
+$(window).click(function() {
+  if (!event.target.matches(".dropbtn")) {
 
-  <h2>BNFC microsite</h2>
-  <iframe src="./example.bnfc.html" class="microsite"></iframe>
-  <iframe src="./example.bnfc.html"></iframe>
+  $(".dropdownContent").each(function() {
+    var openDropdown = $(this);
+    if (openDropdown.hasClass("show")) {
+      openDropdown.removeClass("show");
+    }
+  });
+}});
 
-  <hr/>
+$(document).ready(function() {
 
-  <h2>CKS microsite</h2>
-  <iframe src="./example.cks.html" class="microsite"></iframe>
-  <iframe src="./example.cks.html"></iframe>
+ $('.dropdownContent a').click(function(e) {
+  e.preventDefault();
+  var url = $(this).attr("href");
+  var title = $(this).text();
+  renderIframe(url, title);
+});
 
-  <hr/>
 
-  <h2>Appraisals tool</h2>
-  <iframe src="./example.appraisals.html"></iframe>
-  <iframe src="./example.appraisals.html"></iframe>
+ $(".dropbtn").click(function() {
+  triggerDropdown();
+});
 
-  <hr/>
 
-  <h2>ROI tools</h2>
-  <iframe src="./example.roi.html" class="microsite"></iframe>
-  <iframe src="./example.roi.html"></iframe>
+ function renderIframe(url, title) {
+  var $views = $("#views");
+  if($("#compareTopHats").is(":checked")) {
+  $views.append('<hr>');
+ }
+ else {
+   $views.empty();
+  }
+   $views.append('<h1>' + title + '</h1>');
+   $views.append('<iframe src="' + url + '"></iframe>');
+   $views.append('<iframe class="mobile" src="' + url + '"></iframe>');
+}
 
-  <hr/>
 
-  <h2>InDev tool</h2>
-  <iframe src="./example.indev.html"></iframe>
-  <iframe src="./example.indev.html"></iframe>
+});
 
-  <hr/>
+</script>
+</head>
 
-  <h2>Pathways</h2>
-  <iframe src="./example.pathways.html"></iframe>
-  <iframe src="./example.pathways.html"></iframe>
+<body>
 
+  <section>
+
+    <div class="dropdown">
+
+    <button type="menu" class="dropbtn">Project tophats ⮟ </button>
+      <div id="dropdownList"  class="dropdownContent">
+        <a class="button" href="./example.niceorg.html">Nice org</a>
+        <a class="button" href="./example.guidance.html">Guidence</a>
+        <a class="button" href="./example.standards.html">Standards and guidence</a>
+        <a class="button" href="./example.evidence.html" >Evidence</a>
+        <a class="button" href="./example.evidence-search.html" >Eviedence Search</a>
+        <a class="button" href="./example.bnf.html" >BNF</a>
+        <a class="button" href="./example.bnfc.html">BNFc</a>
+        <a class="button" href="./example.cks.html" >CKS</a>
+        <a class="button" href="./example.appraisals.html" >Appraisals</a>
+        <a class="button" href="./example.roi.html">ROI</a>
+        <a class="button" href="./example.indev.html">In Dev</a>
+        <a class="button" href="./example.pathways.html">Pathways</a>
+      </div>
+
+      <form>
+        <input type="checkbox" id="compareTopHats"> Compare
+      </form>
+    </div>
+  </section>
+  <div id="views"></div>
```

Comment:
An html comment here telling developers that the iframes get rendered inside would be helpful e.g. <div id="views"><!-- Write something here --></div>